### DISCUSSION TITLE
ExampleSite install

### DISCUSSION THREAD

**Discussion by SYSUHPK** [CaiJimmy/hugo-theme-stack#1070] [Q&A]
What are the steps to install a theme from the exampleSite? There seems to be no relevant instructions

**Comment by CaiJimmy** [comment]
> What are the steps to install a theme from the exampleSite?

`exampleSite` is meant for testing purposes. For normal usage, use starter template: https://github.com/CaiJimmy/hugo-theme-stack-starter
